app/web/src/store/secrets.store.ts

```diff
@@ -39,7 +39,7 @@ export function useSecretsStore() {
         ) {
           const id = Math.floor(Math.random() * 899999 + 100000).toString();
 
-          const { id: userId, name: userName } = useAuthStore().user ?? {
+          const { pk: userId, name: userName } = useAuthStore().user ?? {
             id: "-1",
```

Comment:
think you'll want this as `pk` also?

Reply:
I just wanna fix the error, most of this code is going to the backend on my next PR 😄